private/bufpkg/bufcheck/buflint/internal/buflintvalidate/field.go

```diff
@@ -258,7 +258,7 @@ func checkFieldFlags(
 			adder.getFieldRuleName(),
 		)
 	}
-	if fieldConstraints.GetRequired() && fieldConstraints.GetIgnoreEmpty() {
+	if fieldConstraints.GetRequired() && (fieldConstraints.GetIgnoreEmpty() || fieldConstraints.GetIgnore() == validate.Ignore_IGNORE_EMPTY) {
```

Comment:
Kept using the previous deprecated field and also added the new enum with the same semantics. We can remove this when protovalidate reaches v1 and the previous ignore empty boolean is removed. See https://github.com/bufbuild/protovalidate/pull/137 for more details.

cc @rodaine

Reply:
https://github.com/bufbuild/registry-proto/pull/75 means we probably need to update this further to use the non-deprecated name

Reply:
Updated to the latest protovalidate.